tst/functional-tests/framework/fake-servers/control_plane/channel.py

```diff
@@ -0,0 +1,52 @@
+from enum import Enum
+
+class SignalingChanelType(Enum):
+    SINGLE_MASTER = 1
+    FULL_MESH = 2
+
+class SignalingChannelStatus(Enum):
```

Comment:
Can we have description for these statuses? How does this map to the states in our C SDK? I also see we set Status to ACTIVE in `handleCreateSignalingChannelRequest`. Shouldnt the SignalingChannelStatus be set to creating by the peers anywhere or am I understanding this wrong?